network/api.go

```diff
@@ -17,4 +17,6 @@ var (
 	errEndpointNotFound   = fmt.Errorf("Endpoint not found")
 	errEndpointInUse      = fmt.Errorf("Endpoint is already joined to a sandbox")
 	errEndpointNotInUse   = fmt.Errorf("Endpoint is not joined to a sandbox")
+
+	OptVethName = "vethname"
```

Comment:
is there a const file to add this to?

Reply:
we are following the same thing in ipam/api.go